hermes-console-vue/src/views/topic/properties-list/PropertiesList.spec.ts

```diff
@@ -0,0 +1,169 @@
+import { Ack } from '@/api/topic';
+import { describe, expect } from 'vitest';
+import { dummyTopic } from '@/dummy/topic';
+import { render } from '@/utils/test-utils';
+import { within } from '@testing-library/vue';
+import PropertiesList from '@/views/topic/properties-list/PropertiesList.vue';
+import userEvent from '@testing-library/user-event';
+
+describe('PropertiesList', () => {
+  it('should render proper heading', () => {
+    // given
+    const props = { topic: dummyTopic };
+
+    // when
+    const { getByText } = render(PropertiesList, { props });
+
+    // then
+    const row = getByText('topicView.properties.title')!;
+    expect(row).toBeVisible();
+  });
+
+  it.each([
```

Comment:
I don't like putting test cases somewhere else in the code, as reading the test requires u anyway to scroll to those parameters to fully understand the test. If those would be reausable in different tests i would agree but in this case it doesn't give us much.
For example readability of parameterized tests in junit is a disaster because u have to create separate functions providing test cases, and it just super annoying jumping through the code, from test cases to tests and so on